api/services/ocean/aws/paths/cluster-launchSpec.yaml

```diff
@@ -9,7 +9,15 @@ delete:
   parameters:
     - $ref: "../../../../commons/parameters/accountId.yaml"
     - $ref: "../../commons/parameters/oceanLaunchSpecId.yaml"
-    - $ref: "../../commons/parameters/deleteNodes.yaml"
```

Comment:
@vlad666spot is there any other place where this yaml is used?
if this is the only place - lets change the referenced yaml, and not just remove the reference and keep an unused file.

Reply:
@sagishoshan - in reference it is in path params but in this case - it is needed to be in query, so that's why I created inline param. It is used for 'Ocean for GKE' , there it should be also query so I updated deleteNodes.yaml and removed inline param